_includes/docs/latest/resource-coverage.md

```diff
@@ -16,17 +17,16 @@
 | [Compute Engine Instance Templates](https://cloud.google.com/compute/docs/reference/latest/instanceTemplates) |
 | [Firewalls](https://cloud.google.com/compute/docs/reference/latest/firewalls) |
 | [Folders](https://cloud.google.com/resource-manager/reference/rest/v2beta1/folders) |
-| [Folders IAM Policies](https://cloud.google.com/iam/reference/rest/v1/Policy) |
-| [Google Kubernetes Engine Configurations](https://cloud.google.com/kubernetes-engine/docs/reference/rest) |
+| [Folders Cloud IAM Policies](https://cloud.google.com/iam/reference/rest/v1/Policy) |
+| [Kubernetes Engine Configurations](https://cloud.google.com/kubernetes-engine/docs/reference/rest) |
 | [G Suite Groups](https://developers.google.com/admin-sdk/directory/v1/reference/groups) |
 | [G Suite Group Members](https://developers.google.com/admin-sdk/directory/v1/reference/members) |
 | [G Suite Users](https://developers.google.com/admin-sdk/directory/v1/reference/users) |
-| [IAM GrantableRoles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) |
 | [Load Balancer Forwarding Rules](https://cloud.google.com/compute/docs/reference/latest/forwardingRules#resource) |
 | [Networks](https://cloud.google.com/compute/docs/reference/rest/v1/networks) |
 | [Organizations](https://cloud.google.com/resource-manager/reference/rest/v1/organizations) |
-| [Organizations IAM Policies](https://cloud.google.com/iam/reference/rest/v1/Policy) |
+| [Organizations Cloud IAM Policies](https://cloud.google.com/iam/reference/rest/v1/Policy) |
 | [Projects](https://cloud.google.com/resource-manager/reference/rest/v1/projects) |
-| [Projects IAM Policies](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/getIamPolicy) |
+| [Projects Cloud IAM Policies](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/getIamPolicy) |
```

Comment:
See above about names of APIs versus Product names.

Reply:
As above!

Reply:
Resolved.